lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -16,7 +16,9 @@ def module_for(group)
 
       # Only works with RSpec 3.2.0
       def supported?
-        TestProf::Utils.verify_gem_version('rspec', at_least: '3.2.0')
+        required_min_version = '3.2.0'
+        TestProf::Utils.verify_gem_version('rspec', at_least: required_min_version) ||
```

Comment:
Let's just verify `rspec-core` instead, that should work for both Rails and non-Rails projects

Reply:
Hi, @desoleary! Have you seen my comment?